aws/data_source_aws_iam_role_test.go

```diff
@@ -34,6 +34,35 @@ func TestAccAWSDataSourceIAMRole_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDataSourceIAMRole_tags(t *testing.T) {
```

Comment:
Could you also test `resource.TestCheckResourceAttr(dataSourceName, "tags.%", "0"),` in the `TestAccAWSDataSourceIAMRole_basic` case?